packages/config/src/load.js

```diff
@@ -66,5 +78,72 @@ export async function loadNuxtConfig ({
     options.rootDir = rootDir
   }
 
+  // Load env to options._env
+  options._env = env
+
+  // Expand and interpolate runtimeConfig from _env
+  for (const c of ['publicRuntimeConfig', 'privateRuntimeConfig']) {
+    if (options[c]) {
+      expand(options[c], options._env)
+    }
+  }
+
   return options
 }
+
+function loadEnv (rootDir) {
+  const env = Object.create(null)
```

Comment:
Why this and not just `const env = {}` ?

Reply:
Avoid object prototype pollution (null means no proto)